src/grafana_dashboards/postgresql-metrics.json

```diff
@@ -620,7 +622,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "avg(rate(process_cpu_seconds_total{release=\"$release\", instance=\"$instance\"}[5m]) * 1000)",
+          "expr": "avg(irate(process_cpu_seconds_total{release=\"$release\", instance=\"$instance\"}[5m]) * 1000)",
```

Comment:
`irate` allows for better visualizations of spikes in usage of both CPU time (here) and on Memory usage (below).